elytron/src/main/java/org/wildfly/extension/elytron/TlsParser.java

```diff
@@ -332,5 +339,6 @@ public void marshallSingleElement(AttributeDefinition attribute, ModelNode mappi
             .addChild(certificateAuthorityParser)
             .addChild(certificateAuthorityAccountParser)
             .addChild(serverSslSniContextParser)
+            .addChild(dynamicClientSslContextParser) // new
```

Comment:
A new `tlsParser` variable needs to be added and then `dynamicClientSslContextParser` should be added to the new variable instead of here.

Reply:
fixed